js/views/buyWizardVw.js

```diff
@@ -222,37 +223,35 @@ module.exports = baseVw.extend({
   },
 
   showMaps: function(){
-    "use strict";
     this.$el.find('.js-buyWizardMap').removeClass('hide');
-    this.$el.find('.js-buyWizardMapPlaceHolder').removeClass('hide');
-    this.hideMap = false;
   },
 
   hideMaps: function(){
-    "use strict";
     this.$el.find('.js-buyWizardMap').addClass('hide');
-    this.$el.find('.js-buyWizardMapPlaceHolder').addClass('hide');
-    this.hideMap = true;
   },
 
   createNewAddress: function(){
-    "use strict";
     var self = this;
     this.$el.find('.js-buyWizardAddress').addClass('hide');
     this.$el.find('.js-buyWizardNewAddress').removeClass('hide');
     this.$el.find('#buyWizardNameInput').focus();
+    this.$buyWizardMap.find('iframe').addClass('blurMore');
+
     //set chosen inputs
     $('.chosen').chosen();
   },
 
-  hideNewAddress: function(){
+  hideNewAddress: function(e){
     "use strict";
     this.$el.find('.js-buyWizardAddress').removeClass('hide');
     this.$el.find('.js-buyWizardNewAddress').addClass('hide');
+    
+    if (e && $(e.target).hasClass('js-buyWizardNewAddressCancel')) {
```

Comment:
Would it be better to pass in a parameter instead of detecting a click event, so removing the blur could be triggered by other functions (or not triggered if there's a click event that we don't want to trigger it)?